circle.yml

```diff
@@ -1,36 +1,36 @@
-machine:
-  java:
-    version: oraclejdk8
-  environment:
-    TERM: dumb
-  post:
-    # install JCE Unlimited Strength Jurisdiction Policy Files
-    - |
-      wget -qO- --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" "http://download.oracle.com/otn-pub/java/jce/8/jce_policy-8.zip" | bsdtar -xvf- -C /tmp && sudo cp /tmp/UnlimitedJCEPolicyJDK8/* $JAVA_HOME/jre/lib/security/
-
-general:
-  artifacts:
-    - encrypted-config-value/build/libs
-    - encrypted-config-value-bundle/build/libs
-    - encrypted-config-value-bundle-dropwizard1/build/libs
-
-dependencies:
-  override:
-    - ./gradlew resolveConfigurations
-
-test:
-  override:
-    - ./gradlew check
-
-deployment:
-  # disable until we work through OJO publishing issues with Bintray
-  #develop:
-  #  branch: develop
-  #  owner: palantir/encrypted-config-value
-  #  commands:
-  #   - ./gradlew -i artifactoryPublish
-  release:
-    tag: /[0-9]+(\.[0-9]+)+/
-    owner: palantir
-    commands:
-      - ./gradlew -i bintrayUpload
+version: 2
+jobs:
+  build:
+    working_directory: /root/encrypted-config-value
+    environment:
+      ARTIFACTS_DIR: /tmp/artifacts
+      TESTS_DIR: /tmp/tests
+    docker:
+      - image: nmiyake/java:alpine-oracle-jdk-8-t3
+        cmd: ["/bin/bash"]
+    steps:
+      - type: checkout
+      - type: cache-restore
+        key: gradle-{{ checksum "/root/encrypted-config-value/gradle/wrapper/gradle-wrapper.properties" }}-v1
+      - type: shell
+        name: "Resolve Gradle configurations"
+        command: ./gradlew resolveConfigurations
+      - type: cache-save
+        key: gradle-{{ checksum "/root/encrypted-config-value/gradle/wrapper/gradle-wrapper.properties" }}-v1
+        paths:
+          - /root/.gradle
+      - type: shell
+        name: "Run checks, build and copy artifacts to artifacts directory"
+        command: ./gradlew check build copyCiArtifactsToCircle
+      - type: test-results-store
+        path: /tmp/tests
+      - type: artifacts-store
+        path: /tmp/artifacts
+      - type: deploy
+        name: "Publish on release tags"
```

Comment:
Yup -- looks like it's on their radar, but based on https://discuss.circleci.com/t/build-on-tag/9864/31 (which is the main thread discussing this), last update was 20 days ago and was not yet available (they said they would update the thread when it is available).